assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -115,13 +113,13 @@ che.workspace.sidecar.default_memory_request_mb=64
 
 # CPU limit default for each sidecar that has no CPU settings in the {prod-short} plug-in configuration.
 # Specify either in floating point cores number, for example, `0.125`,
-# or using the Kubernetes format, integer millicores, for example, `125m`.
+# or using the {orch-name} format, integer millicores, for example, `125m`.
```

Comment:
Openshift is Kubernetes and this format is defined by Kubernetes specification. I would not change it.

Reply:
So "Kubernetes format" in downstream docs?